app/services/doc_auth/vendors.rb

```diff
@@ -0,0 +1,8 @@
+module DocAuth
+  module Vendors
```

Comment:
> Any thoughts on moving these into `IdP::Constants`? It adds a prefix to the name which may get a _little_ unwieldy but it does keep them all together.

Yeah, we could do that. One issue we may run into if we use the top-level vendors is that we refer to LexisNexis as [`'lexis_nexis'` in the context of PhoneFinder](https://github.com/18F/identity-idp/blob/a294bd6096b54a400be519baa9c7a2498aca5b7a/app/services/idv/steps/verify_base_step.rb#L17), but [`'lexisnexis'` in the context of TrueID](https://github.com/18F/identity-idp/blob/a294bd6096b54a400be519baa9c7a2498aca5b7a/app/services/doc_auth_router.rb#L171). 😬 (Constant-izing this earlier might have helped that 😅 )

Reply:
Looking at this more closely, I think we can probably update references to "lexisnexis" to "lexis_nexis", since this value is controlled through configuration (`doc_auth_vendor` and `doc_auth_vendor_randomize_alternate_vendor`). Since we're only using TrueID in some non-production environments, will just need to update them accordingly.

Reply:
Based on [a related Slack discussion](https://gsa-tts.slack.com/archives/C0NGESUN5/p1657655307643849), I'm going to hold off on the rename for now. In the meantime, I'll move these values into `Idp::Constants`, but use `'lexisnexis'` for LexisNexis. The follow-on pull request will rename that to `'lexis_nexis'` and update the reference in `VerifyBaseStep`.

Reply:
Follow-up pull request at #6581.